### PR TITLE
Update Unit Tests for FreeRTOS IP Timers

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
Update Unit Tests for FreeRTOS IP Timers

Description
-----------
Fix Unit Tests for FreeRTOS IP Timers

Test Steps
-----------
Unit Tests Build and Run steps

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kar-rahul-aws** [comment]
bot run uncrustify

File: test/unit-test/FreeRTOS_IP_Timers/FreeRTOS_IP_Timers_utest.c [hunk 1]
```diff
@@ -289,17 +289,20 @@ void test_vCheckNetworkTimers_ARPResolutionTimerActiveAndExpired2( void )
 
 void test_vCheckNetworkTimers_DHCPTimerActiveAndExpired( void )
 {
+    struct xNetworkEndPoint pxEndPoint;
```
Comment (moninom1) [inline test/unit-test/FreeRTOS_IP_Timers/FreeRTOS_IP_Timers_utest.c:292]: We can rename it to xEndPoint as this is not a pointer

File: test/unit-test/FreeRTOS_IP_Timers/FreeRTOS_IP_Timers_utest.c [hunk 2]
```diff
@@ -289,17 +289,20 @@ void test_vCheckNetworkTimers_ARPResolutionTimerActiveAndExpired2( void )
...
+    pxEndPoint.pxNext = NULL;
+    
     xARPTimer.bActive = pdFALSE;
-    xDHCPTimer.bActive = pdTRUE;
+    //xDHCPTimer.bActive = pdTRUE;
```
Comment (moninom1) [inline test/unit-test/FreeRTOS_IP_Timers/FreeRTOS_IP_Timers_utest.c:296]: We can remove it if not needed.

**Comment by kar-rahul-aws** [comment]
Unit Tests have been updated in dev/IPv6 integration branch

**Pull request closed without merging**